### PR TITLE
Search by key,location, jobType, skills

### PR DESCRIPTION
**Pull request by meriem26**

### CODE REVIEW

File: client/src/components/OpportunitiesList.js [hunk 1]
```diff
@@ -90,24 +89,81 @@ class OpportunitiesList extends Component {
     });
   };
   handleSelectCity = (e, data) => {
-    const selectedCity = data.value;
+    const City = data.value;
```
Comment (kabaros) [inline client/src/components/OpportunitiesList.js]: make it small letter `city` not `City` .. and try to use destructuring for this

File: client/src/components/OpportunitiesList.js [hunk 2]
```diff
@@ -90,24 +89,81 @@ class OpportunitiesList extends Component {
...
+          }).length
+        );
+      });
+    }
+
+    return opportunitiesList;
+  };
   render() {
-    console.log(this.state.selectedJobType);
+    console.log(this.state.selectedCity);
```
Comment (kabaros) [inline client/src/components/OpportunitiesList.js]: remove console.log

File: client/src/components/OpportunitiesList.js [hunk 3]
```diff
@@ -90,24 +89,81 @@ class OpportunitiesList extends Component {
...
+        );
+      });
+    }
+    if (selectedCity && !selectedJobType) {
+      return opportunitiesList.filter(opportunity => {
+        return selectedCity.includes(opportunity.location);
+      });
+    }
+    if (!selectedCity && selectedJobType) {
+      return opportunitiesList.filter(opportunity => {
```
Comment (kabaros) [inline client/src/components/OpportunitiesList.js]: if the arrow function has only one statement, then you can drop the `{` `}` and the `return`

Comment (kabaros) [reply client/src/components/OpportunitiesList.js]: same for other ones in this file

File: client/src/utils/constants.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
 export const opportunityType = [
-  { key: "1", text: "voluntary work", value: "voluntary work" },
-  { key: "2", text: "training", value: "training" },
-  { key: "3", text: "internship", value: "internship" },
-  { key: "4", text: "work experience", value: "work experience" },
-  { key: "5", text: "part time job", value: "part time job" },
-  { key: "6", text: "full time job", value: "full time job" }
+  { key: "1", text: "Volunteer", value: "Volunteer" },
```
Comment (kabaros) [inline client/src/utils/constants.js]: why do you need `key`? you can use `value` as the key `.map` as it is unique

Comment (youssef910) [reply client/src/utils/constants.js]: to insert the array to the dropdpwn as option should be in that form
https://react.semantic-ui.com/modules/dropdown/#types-selection

File: client/src/components/OpportunitiesList.js (hunk 2 above)
Comment (meriem26) [reply client/src/components/OpportunitiesList.js]: done

**Pull request closed without merging**